app/models/levelling_up_payments.rb

```diff
@@ -1,3 +1,11 @@
 module LevellingUpPayments
   extend self
+
+  def short_name
+    I18n.t("levelling_up_payments.policy_short_name")
+  end
+
+  def routing_name
```

Comment:
I think this goes in `Journey` now.

Reply:
I've pasted something into all the policies to look into the `Journey` instead. Unfortunately `routing_name` is called all over the place so there's not a quick way of tidying up.